test/envoye2e/stackdriver_plugin/stackdriver_test.go

```diff
@@ -842,63 +842,88 @@ func TestStackdriverRbacAccessDenied(t *testing.T) {
 	respCode := "403"
 	logEntryCount := 5
 
-	params := driver.NewTestParams(t, map[string]string{
-		"ServiceAuthenticationPolicy": "NONE",
-		"DirectResponseCode":          respCode,
-		"SDLogStatusCode":             respCode,
-		"StackdriverRootCAFile":       driver.TestPath("testdata/certs/stackdriver.pem"),
-		"StackdriverTokenFile":        driver.TestPath("testdata/certs/access-token"),
-		"RbacAccessDenied":            "true",
-		"RbacDryRun":                  "true",
-	}, envoye2e.ProxyE2ETests)
+	rbacCases := []struct {
+		name             string
+		rbacDryRunKey    string
+		rbacDryRunFilter string
+	}{
+		{
+			name:             "ActionBoth",
+			rbacDryRunKey:    "RbacDryRunDenied",
```

Comment:
nit: it is a bit weird to specify the key instead of the value here. Can you instead make the value the variable? Like make the key `bacDryRunResult`.

Reply:
sure, updated.